src/discretization/fist.jl

```diff
@@ -142,3 +144,15 @@ function isearccw(𝒫::Chain{Dim,T}, i) where {Dim,T}
 
   isconvex && !intersects && incones
 end
+
+function remove_adjacent_ears!(𝒬, i)
+  ind = 1
+  while ind <= length(𝒬)
```

Comment:
Instead of traversing all the ears, assuming they are ordered, we can just check for the i-2 and i+2 ones.

Reply:
I am confused because the original algorithm doesn't treat the neighbor ears immediately after the pop, but we can discuss over a call today the details.